cleanEnv.js

```diff
@@ -25,28 +25,36 @@ async function clearLogFiles(directory) {
   }
 }
 
-function deleteBitcoinDataDirectory() {
+function clearBitcoinDataDirectory() {
```

Comment:
Refactored this utility to delete all files and directory inside the bitcoin data directory, but not the bitcoin data directory itself.